src/display/api.js

```diff
@@ -1650,7 +1650,9 @@ const PDFWorker = (function PDFWorkerClosure() {
       }
       if (typeof PDFJSDev === "undefined" || !PDFJSDev.test("PRODUCTION")) {
         if (typeof SystemJS !== "object") {
-          throw new Error("SystemJS must be used to load fake worker.");
+          // Manually load SystemJS, since it's only necessary for fake workers.
+          await loadScript("../node_modules/systemjs/dist/system.js");
```

Comment:
Shouldn't this be `../../` instead of `../` since, contrary to the same change in the `web` folder, this file is inside another subfolder?

Reply:
I'd be inclined to agree with you here, but interestingly the patch as written actually works (when this code-path is forced, since the development viewer won't generally hit it).

The explanation would rather be that when this file is loaded, it's done through the `web/viewer.html` file which does seem to end up being the effective "scope" used here.

Finally, this is fallback code which should *never* be hit in development mode and this is only necessary until https://bugzilla.mozilla.org/show_bug.cgi?id=1247687 is fixed.